kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/api.scala

```diff
@@ -39,18 +40,19 @@ private[api] trait ApiRequestContext {
 }
 
 @Provider
-class RestExceptionMapper extends ExceptionMapper[Exception] {
+class RestExceptionMapper extends ExceptionMapper[Exception] with Logging {
   override def toResponse(exception: Exception): Response = {
+    warn("Error occurs on accessing REST API.", exception)
```

Comment:
`trino.api` also has a similar `RestExceptionMapper`
Can it be updated together?

https://github.com/apache/kyuubi/blob/c5238535f6c8d70ac97efc4c1d5dee93765fef92/kyuubi-server/src/main/scala/org/apache/kyuubi/server/trino/api/api.scala#L42-L45

Reply:
Thanks, updated